_posts/2025-08-19-cockpit-345.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Cockpit 345
+author: mvo
+date: '2025-08-19'
+tags: cockpit, machines
+slug: cockpit-345
+category: release
+summary: 'IPv6 for Wireguard, detachable VNC console'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).
+We release regularly.
+
+Here are the release notes from Cockpit 345 and cockpit-machines 338:
+
+
+## Networking: Support IPv6 addresses for WireGuard
+
+The Networking page can now configure IPv6 addresses for WireGuard VPN
+directly within the WireGuard modal.
+
+<img width="840" height="684" alt="Screen Shot 2025-08-20 at 14 16 46" src="https://github.com/user-attachments/assets/9453ee37-12b8-4882-b8dd-b638cdc1b606" />
+
+## Machines: Allow pop-out of VNC console viewer
+
+You can now detach the VNC console viewer of a virtual machine into
+its own window.
+
+<img width="863" height="141" alt="image" src="https://github.com/user-attachments/assets/f12932a7-c0da-40a3-a5a7-e144b604fac4" />
+
+## Try it out
+
+Cockpit 345 and cockpit-machines 338 are available now:
+
+* [For your Linux system](https://cockpit-project.org/running.html)
+* [Cockpit Client](https://flathub.org/apps/details/org.cockpit_project.CockpitClient)
+
+* [Cockpit Source Tarball](https://github.com/cockpit-project/cockpit/releases/tag/345)
+* [Cockpit Fedora 42](https://bodhi.fedoraproject.org/updates/?releases=F42&packages=cockpit)
+* [Cockpit Fedora 41](https://bodhi.fedoraproject.org/updates/?releases=F41&packages=cockpit)
+* [cockpit-machines Source Tarball](https://github.com/cockpit-project/cockpit-machines/releases/tag/338)
+* [cockpit-machines Fedora 42](https://bodhi.fedoraproject.org/updates/?releases=F42&packages=cockpit-machines)
+* [cockpit-machines Fedora 41](https://bodhi.fedoraproject.org/updates/?releases=F41&packages=cockpit-machines)
```

Comment:
Links can be updated now :)

Reply:
We don't do that anymore, I thought. This step has been removed from the release process.